src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java

```diff
@@ -26,9 +26,9 @@
   static {
     SLOT_SELECTORS = new HashMap<>();
     SLOT_SELECTORS.put(TOP_LEADERBOARD, "div[id$='TOP_LEADERBOARD_0__container__']");
-    SLOT_SELECTORS.put(BOTTOM_LEADERBOARD, "div[id$='BOTTOM_LEADERBOARD_0__container__']");
-    SLOT_SELECTORS.put(BOTTOM_LEADERBOARD_DESKTOP, "div[id$='BOTTOM_LEADERBOARD_0__container__']");
-    SLOT_SELECTORS.put(BOTTOM_LEADERBOARD_MOBILE, "div[id$='BOTTOM_LEADERBOARD_0__container__']");
+    SLOT_SELECTORS.put(BOTTOM_LEADERBOARD, "div[id*='wka1a.PF/bottom_leaderboard'][id*='_0__container__']");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "div[id*='wka1a.PF/bottom_leaderboard'][id*='_0__container__']" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)